.tree-sitter-lint/tree-sitter-lint-local/src/bin/tree-sitter-lint-local.rs

```diff
@@ -1,3 +1,11 @@
+use tracing_chrome::ChromeLayerBuilder;
+use tracing_subscriber::prelude::*;
+
 fn main() {
+    // tracing_subscriber::fmt::init();
+
+    let (chrome_layer, _guard) = ChromeLayerBuilder::new().include_args(true).build();
+    tracing_subscriber::registry().with(chrome_layer).init();
```

Comment:
It's silly that this is committed as "always on" but this was pretty awesome how easy this was to get nice timing visualizations

Maybe I should make this activated via an environment variable?